src/main/scala/com/github/bzumhagen/sct/ChangelogConfiguration.scala

```diff
@@ -14,7 +14,7 @@ object ChangelogConfiguration {
       referencePattern = config.getString("sct.patterns.reference").r,
       tags = config.getStringList("sct.tags").asScala.toSet,
       smartGrouping = config.getBoolean("sct.smartGrouping"),
-      showReference = config.getBoolean("sct.showReference")
```

Comment:
I think we should maintain showreference option, for example if exposing to customers they aren't going to care about the ticket reference to our internal system which they don't have access to.

Reply:
good point, I'll bring it back